napari/_qt/menus/plugins_menu.py

```diff
@@ -34,7 +34,7 @@ def __init__(self, window: 'Window'):
 
     def _build(self, event=None):
         self.clear()
-        action = self.addAction(trans._("Install/Uninstall Plugins..."))
+        action = self.addAction(trans._("napari Plugin Manager"))
```

Comment:
I personally think it's weird to have `napari Plugin Manager` from within `napari` - like, who else's would it be 😆 . I think this should just be `Plugin Manager` - happy to be overridden though! I know we're thinking about splitting this out into its own package etc.

Reply:
I went ahead and removed `napari`. I will put it back if anyone objects.